web/app/styles/chinese-checkers.less

```diff
@@ -1,4 +1,17 @@
 @import "_fonts";
+@import "_colors";
 @import "_common";
+@import "_icons";
+
+@import "views/_game-view";
 
 @import "components/_app";
+@import "components/_card";
```

Comment:
Each component should import its own CSS file, instead of having to merge them all together here